src/util.js

```diff
@@ -356,6 +356,18 @@ export function getFirstAnnotation(annotations) {
     return annotations[firstAnnotationId];
 }
 
+/**
+ * Return first annotation in thread
+ *
+ * @param {Object} annotations - Annotations in thread
+ * @return {Annotation} First annotation in thread
+ */
+export function getLastAnnotation(annotations) {
+    const numAnnotations = Object.keys(annotations).length;
+    const lastAnnotationID = Object.keys(annotations)[numAnnotations - 1];
```

Comment:
can you bail out early if there are no annotations?

Reply:
Not sure if it's necessary in this or the `getFirstAnnotation()` method since neither are costly calls. Curious as to the reason why we should be bailing out early?